test/spec.sh

```diff
@@ -290,10 +290,14 @@ sh-func() {
 
 glob() {
   # Note: can't pass because it assumes 'bin' exists, etc.
-  sh-spec spec/glob.test.sh --oils-failures-allowed 4 \
+  sh-spec spec/glob.test.sh --oils-failures-allowed 3 \
```

Comment:
Oh also if you want something "boring" to do I would not mind if all these go inside the spec files

I added the `## oils_failures_allowed` lines a few months ago, and not all spec tests have been converted

Ideally I want to get rid of all these tiny functions -- that is a "refactoring in flight"

But it's not urgent, something to do in the background